### ISSUE TITLE
Add editable fields for streets

### ISSUE THREAD

**Issue by n-peugnet** [organicmaps/organicmaps#1323]
I wanted to Mark a street as a One-way street but there was no editable fields when I tried to edit the street.

I don't know how it is stored on the OSM end but I think this would be an important data to edit, as it would enhance the navigation experience by not trying to use streets in the opposite direction.

Also this may require some UI work to be sure that the direction selected is the right one.

**Comment by biodranik** [comment]
Editing streets is dangerous because non-experienced users may easily break the map. I think it would be safer to leave a note for OSMers.

**Comment by endim8** [comment]
Agreed, an incorrectly tagged road can mess routing and stuff up _a lot_

**Comment by n-peugnet** [comment]
Ok I understand. Maybe a message in the edition page explaining just what you said could help people understand why it is not possible to edit streets?

Feel free to close this issue whenever you want.

**Comment by keunes** [comment]
Maybe structured notes could be created? E.g. have 'property' and 'value' fields in the UI, which are turned into a note saying something like "It is suggested to add/change property X to/with value Y"

**Comment by biodranik** [comment]
This is an important issue to improve the road graph. I know it is complex, and it is easy to break OSM data, but we need to have an easy way to report issues, or maybe to allow _advanced_ OSM users with a proven history of edits to fix data directly. Let's brainstorm on the UX of this feature.
